.github/workflows/push-to-testnet.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        sbc: [raspi, rockpi, rak, finestra, sensecap, og, controllino, pisces, cotx, pantherx1]
+        variant: [nebra-indoor1, nebra-outdoor1, nebra-indoor2, nebra-light1, nebra-light2, nebra-light3]
```

Comment:
There should be outdoor 2 in all of them. We don't have it ready yet but it should be there